framework/core/src/Extension/Console/ToggleExtensionCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Extension\ExtensionManager;
+
+class ToggleExtensionCommand extends AbstractCommand
+{
+    /**
+     * @var ExtensionManager
+     */
+    protected $extensionManager;
+
+    public function __construct(ExtensionManager $extensionManager)
+    {
+        $this->extensionManager = $extensionManager;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('extension:toggle')
+            ->setDescription('Enable or disable an extension.')
+            ->addArgument('extension', null, 'The extension to enable or disable.')
+            ->addOption('enable', null, null, 'Enable the extension.')
+            ->addOption('disable', null, null, 'Disable the extension.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function fire()
+    {
+        $name = $this->input->getArgument('extension');
+
+        if ($this->input->getOption('enable')) {
+            $this->info("Enabling $name extension...");
+            $this->extensionManager->enable($name);
+        } elseif ($this->input->getOption('disable')) {
+            $this->info("Disabling $name extension...");
+            $this->extensionManager->disable($name);
+        } else {
+            $this->error('You must specify --enable or --disable.');
+        }
```

Comment:
We can then check which command name was used and effectively drop the `else`.
we also have to check if the extension exists before doing anything otherwise we get exceptions.
An additional nicety imo, would be to be explicit about an extension being already enabled or disabled 😸 

```suggestion
        $name = $this->input->getArgument('extension-id');
        $enabling = $this->input->getFirstArgument() === 'extension:enable';

        if ($this->extensionManager->getExtension($name) === null) {
            $this->error("There are no extensions by the ID of '$name'.");
            return;
        }

        switch ($enabling) {
            case true:
                if ($this->extensionManager->isEnabled($name)) {
                    $this->info("The '$name' extension is already enabled.");
                    return;
                } else {
                    $this->info("Enabling '$name' extension...");
                    $this->extensionManager->enable($name);
                }
                break;
            case false:
                if (! $this->extensionManager->isEnabled($name)) {
                    $this->info("The '$name' extension is already disabled.");
                    return;
                } else {
                    $this->info("Disabling '$name' extension...");
                    $this->extensionManager->disable($name);
                }
                break;
        }
```